assets/js/types/type-defs/payments.ts

```diff
@@ -91,6 +91,16 @@ export type PaymentMethods =
 	| Record< string, PaymentMethodConfigInstance >
 	| EmptyObjectType;
 
+/**
+ * Used to represent payment methods in a context where storing objects is not allowed, i.e. in data stores.
+ */
+export type PlainPaymentMethods = Record< string, { name: string } >;
+
+/**
+ * Used to represent payment methods in a context where storing objects is not allowed, i.e. in data stores.
+ */
+export type PlainExpressPaymentMethods = PlainPaymentMethods;
```

Comment:
Nitpick: I see that all other types do not have comments, so I wonder if we need to comment these two types. If so, I'd probably use a single-line comment instead of a multiline comment to save save.